benchmark/Streamly/Benchmark/Prelude/WSerial.hs

```diff
@@ -193,9 +193,13 @@ o_1_space_concat value =
             "concatMapWithInterleave (n of 1)"
             (concatStreamsWith Internal.interleave value 1)
         -}
+
+        -- XXX stack overflow with --long
```

Comment:
Was this working in previous release?

Reply:
interleave and roundrobin are unlikely to work with large number of streams, definitely not with infinite number of streams, therefore they should be re-classified to o-n-space. I am wondering how it was passing in earlier releases? We also have this comment above:

-- concatMapWith using StreamD versions of interleave operations are
-- all quadratic, we just measure the sqrtVal benchmark for comparison.

Reply:
They were not working with the previous release either.